### PR TITLE
New cookbook for sinopia-github which adds github authentication to sinopia

### PR DESCRIPTION
**Pull request by gilleyj**
hacky but it works

### CODE REVIEW

**Comment by gilleyj** [comment]
I have a deploy.json that has the github auth keys, let me know if anyone needs this to test in /cookbooks/vagrant-test/sinopia/

**Comment by gilleyj** [comment]
This go round is NOT so hacky at all.

File: sinopia-github/attributes/default.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+# limitations under the License.
+#
+
+# encoding: utf-8
+
+## System user running sinopia-github
+default['sinopia-github']['user'] = 'sinopia-github'
+
+## sinopia-github gem version (use nil for latest)
+default['sinopia-github']['version'] = nil
```
Comment (till) [inline sinopia-github/attributes/default.rb]: Not used, right?

Comment (gilleyj) [inline sinopia-github/attributes/default.rb]: Fixed.  And by fixed I mean removed.

**Comment by gilleyj** [comment]
yay

**Comment by till** [comment]
:rocket:

**Pull request merged**